glue_jupyter/bqplot/image/state.py

```diff
@@ -0,0 +1,13 @@
+from echo import CallbackProperty
+
+from glue.viewers.image.state import ImageViewerState
+
+
+class BqplotImageViewerState(ImageViewerState):
+
+    # TODO: change contour_visible to False by default once it can be toggled in UI
+
+    bitmap_visible = CallbackProperty(True, 'whether to show the image as a bitmap')
+    contour_visible = CallbackProperty(True, 'whether to show the image as contours')
+
+    mode = CallbackProperty('image', 'Whether to show an image or contour')
```

Comment:
I think it makes more sense to have this in the Layer state, and having both mode and the two visibilities sounds a bit duplicate right?

Reply:
Oops yes I intended to have this in the layer state, and indeed ``mode`` is left over from a previous experiment